src/sqlRunner.ts

```diff
@@ -94,7 +95,10 @@ export function extractSqlFileInfo(filePath: string): SqlFileInfo {
 export function getDropStatement(type: string, fqon: string): string {
   const dropStatement = dropStatementsMap[type];
 
-  return dropStatement ? `${dropStatement} ${fqon}` : '';
+  if (dropStatement) { return `${dropStatement} ${fqon}`; }
```

Comment:
I'd like this to be the other way round so that it is clear what the function is returning.

```js
if (!dropStatement) {
  const message = `Naming convention must be exact as the directory names inside src/`;
  
  throw new Error('No database object type: ${type}. ${message}');
}

return `${dropStatement} ${fqon}`;
```